external/easylogging++/easylogging++.h

```diff
@@ -1079,7 +1080,7 @@ namespace el {
                                 /// @brief Gets ID of currently running threading using std::this_thread::get_id()
                                 static inline std::string getCurrentThreadId(void) {
                                     std::stringstream ss;
-                                    ss << std::this_thread::get_id();
+                                    ss << std::setfill(' ') << std::setw(16) << std::hex << std::this_thread::get_id();
```

Comment:
Referencing https://github.com/easylogging/easyloggingpp/pull/432 in case anyone else asks if this was PR'd upstream 😄